### ISSUE TITLE
blank page when running

### ISSUE THREAD

**Issue by SolitudePy** [volatilityfoundation/volumetric#4]
Hello, tried to make it work and after I have done all of the steps I result in a blank page, the node_modules directory did appear under volumetric-master and also under resources, the get requests seems okay but nothing happens.
also volumetric calls for package named volatility while its named volatility3 which causes a lot of issues...

**Comment by ikelos** [comment]
Hiya, thanks very much, you're quite right, the codebase hadn't been updated in quite a while.  I've now bumped everything to point to the right package name and also fixed a minor issue (which means that this now requires volatility3 version 2.5.2, which is currently still only in the development branch).  I've tested this and doesn't just provide a white page.  If you're still running into the issue, please feel free to reopen it and we can start debugging from there...  5:)

**Comment by SolitudePy** [comment]
@ikelos still a blank page sadly, I do not see errors in the console

**Comment by ikelos** [comment]
Did you make sure to follow [the installation guide](https://github.com/volatilityfoundation/volumetric?tab=readme-ov-file#install-webcomponent-dependencies) and install the web components using npm?  If they're installed then you should be served *something*?  Otherwise you will be returned a blank page because the tool hasn't been installed correctly...

**Comment by SolitudePy** [comment]
@ikelos  yeah... `npm list` retrieves a correct list of dependencies, the server is able to run but a blank page is met. the requirements also installed successfully. Ctrl + U on the blank page shows me the the index.html html file. I also tried different browsers

**Comment by ikelos** [comment]
Ok, could you please attach the output of the server after you start it and make a request, and then we can compare it to a working version and see where they deviate.  You might also want to see if your browser console lists an issue...

**Comment by SolitudePy** [comment]
Devconsole seems okay. here is the output https://pastebin.com/Ss3wNSb6

**Comment by ikelos** [comment]
Ok, so then if you can go to the web tools and look at the Elements, volumetric-shell should have a #shadow-root object underneath it, and when you mouse over it, it should show you the size/height of the element on the page.  You web log shows that the web components are getting pulled down, and if your web console is empty it suggests no errors, so I'm at a bit of a loss.  You could check the network tab and make sure all the requests are succeeding?

**Comment by SolitudePy** [comment]
I got 1536x8 from hovering the `<html lang="en">`, and the requests in Network tab all succeed with 200 status code. im quite at a loss

**Comment by ikelos** [comment]
Ok, so you do get a `volumetric-shell` element, and if so how big is it?  Is there a `#shadow-root` under it?

**Comment by SolitudePy** [comment]
oh u meant the `<volumetric-shell>`, there isnt a `#shadow-root` under it and its 0x0 size

**Comment by SolitudePy** [comment]
![image](https://github.com/volatilityfoundation/volumetric/assets/47316655/c70bfee2-b072-4a30-8bce-00e7282b74ed)

**Comment by ikelos** [comment]
Ok, then something's up with web components (although it did seem to try grabbing most of them?).  Do you have javascript disabled or any other unusual settings?  Could you try it in another browser and/or a private browsing window to see if there's something interfering with it?

**Comment by SolitudePy** [comment]
I tried different browsers, incognito, etc already.

**Comment by ikelos** [comment]
I'm afraid this is sounding more and more like a problem specific to the setup?  Do you have any other machines, or a virtual machine, you could configure to try it out?  The pages seem to be transferred correctly, but aren't being interpreted by your browser as web components, but web components have been standardized for years...
